docs/MPURegionsUsageGuide.md

```diff
@@ -30,41 +40,48 @@
 ```
 
 ## How to control MPU
-1. **How to add new MPU region**
-	- Logical mapping of MPU regions to their respective memory segments is done statically using below enumeration in common mpu.h header file (os/include/tinyara/mpu.h).
+1. **Board specific MPU region reservation**
+	- In order to support few regions for the board specific purpose, below API is provided to reserve the first *num* number of regions:
+	```c
+	void mpu_set_nregion_board_specific(uint8_t num);
+	```
+	Above function shall be called during board initialization after the board specific MPU regions have been configured. The TizenRT MPU regions are configured after *num* number of board MPU regions.
+
+2. **MPU usage overview**
```

Comment:
I think this should be at 1, as it is overview and later board specific reservation will come